hypershift-operator/controllers/hostedcluster/hostedcluster_controller.go

```diff
@@ -450,6 +450,7 @@ func reconcileHostedControlPlane(hcp *hyperv1.HostedControlPlane, hcluster *hype
 		Name: fmt.Sprintf("%s-kubeconfig", hcluster.Spec.InfraID),
 		Key:  "value",
 	}
+	hcp.Spec.Services = hcluster.Spec.Services
```

Comment:
Are these actually mutable?

Reply:
"Mutable" down the line as if there's another service that needs to be exposed in the control plane it can be accounted for here.

Not "mutable" though currently as you can dynamically shift node ports.

Reply:
In the standard case: these will remain mostly the same though but there are possibilities that they can change.